lib/nostrum/api.ex

```diff
@@ -99,22 +99,23 @@ defmodule Nostrum.Api do
     - `pid` - Pid of the shard.
     - `status` - Status of the bot.
     - `game` - The 'playing' text of the bot. Empty will clear.
+    - `type` - The type of status to show.
     - `streaming` - URL of twitch.tv stream
```

Comment:
This should be `stream`, my mistake, might as well fix it while you're in here. :^).